SPECS/uefistored.spec

```diff
@@ -42,9 +42,14 @@ make
 # symlink binary to varstored to let XAPI find it
 ln -s uefistored %{buildroot}%{_sbindir}/varstored
 
+# Install PK.auth
+mkdir -p %{buildroot}%{_prefix}/share/varstored/
+cp %{_sourcedir}/PK.auth %{buildroot}%{_prefix}/share/varstored/PK.auth
```

Comment:
I did not know you could even do that in a RPM spec file!

However I need to ask you to replace this with a Source1 definition at line 8 and %{SOURCE1} here :)